src/Themes/NexusMods.Themes.NexusFluentDark/Resources/Palette/Colors/BrandColors.axaml

```diff
@@ -244,13 +243,4 @@
     <SolidColorBrush x:Key="BrandTranslucentDark100Brush" Color="{StaticResource BrandTranslucentDark100}" />
     <SolidColorBrush x:Key="BrandTranslucentDark50Brush" Color="{StaticResource BrandTranslucentDark50}" />
 
-
-    <!-- Primary Legacy -->
-    <!-- NOTE: These are the legacy Primary colours, and should be removed once the new Primary colours are ready to be used. -->
-    <StaticResource x:Key="BrandLegacyPrimaryWeak" ResourceKey="PrimitiveLegacyPrimaryWeak" />
-    <StaticResource x:Key="BrandLegacyPrimaryModerate" ResourceKey="PrimitiveLegacyPrimaryModerate" />
-    <StaticResource x:Key="BrandLegacyPrimaryStrong" ResourceKey="PrimitiveLegacyPrimaryStrong" />
```

Comment:
This is still used by `MarkdownRendererView.axaml`.

Reply:
Found it, thanks